source/adapters/level_zero/platform.cpp

```diff
@@ -320,6 +320,22 @@ ur_result_t ur_platform_handle_t_::initialize() {
                 &ZeMutableCmdListExt
                      .zexCommandListUpdateMutableCommandWaitEventsExp))) == 0);
 
+  ZeMutableCmdListExt.Supported &=
```

Comment:
Noting that this conflicts with https://github.com/oneapi-src/unified-runtime/pull/2351, not that I think there's any action to do yet but address conflicts.